karaage/people/forms.py

```diff
@@ -92,6 +92,17 @@ class Meta:
             'address', 'country'
         ]
 
+    def clean_email(self):
+        email = self.cleaned_data['email']
+        users = Person.objects.filter(email__exact=email)
+        if users.count() > 0:
+            raise forms.ValidationError(
+                six.u(
+                    'An account with this email already exists. '
+                    'Please email %s')
+                % settings.ACCOUNTS_EMAIL)
```

Comment:
I don't think the above code will work as is. The problem is when updating a person, it will force you to change the email address every time, as I think it will detect that email address is already in use by the person you are editing. Might be why the tests are failing, not sure.

Reply:
Updated and merged in b70ea155.

Reply:
Note that the tests were correctly failing - we were attempting to create a user with an email address that already existed in the system.